### DISCUSSION TITLE
Help Please. Windows x 64

### DISCUSSION THREAD

**Discussion by cmbconcretefire** [comit-network/xmr-btc-swap#690] [Q&A] [answered]
I downloaded both 0.8.1 latest release files for win x 64. ASB.exe I think and swap.exe......... that is literally the only 2 files that show up when I unzip the packages. 

When I double click on either one of them, nothing happens. Is there tutorial for beginners? Or am I missing something?

**Comment by thomaseizinger** [comment, marked as answer]
The tool needs to be launched from the terminal. The docs are here: https://github.com/comit-network/xmr-btc-swap/tree/master/docs/cli

**Reply by cmbconcretefire** [reply]
Thank you for the response Thomas but without a GUI, the thing is useless. This is 2021, not 1978. Requesting a GUI please. 
I'll check back in a couple years. 

Yes you pointed me to the CLI document but I don't know what CLI is or where to get it. This product isn't ready for mainstream and didn't deserve the hype.

**Reply by thomaseizinger** [reply]
What if I told you that even in 2021, there is an entire world of software that doesn't have a GUI and is certainly not useless :)

There is an ongoing discussion [here](https://github.com/comit-network/xmr-btc-swap/discussions/654) on how what we built can be utilized to build a more end-user friendly product with a GUI.

**Reply by nanostos** [reply]
> Thank you for the response Thomas but without a GUI, the thing is useless. This is 2021, not 1978. Requesting a GUI please.
> I'll check back in a couple years.
> 
> Yes you pointed me to the CLI document but I don't know what CLI is or where to get it. This product isn't ready for mainstream and didn't deserve the hype.

Just follow the point-and-click prompts via unstoppableswap.net and it's really quite easy.

**Comment by cmbconcretefire** [comment]
Ok. Thanks.
